src/components/UserInfo/UserInfo.jsx

```diff
@@ -1,3 +1,7 @@
-export const UserInfo = () => (
-  <>Put the user here</>
+import './UserInfo.scss';
+
+export const UserInfo = ({ user }) => (
+  <a className="UserInfo" href={`mailto:${user.email}`}>
+    {user && user.name}
```

Comment:
Potential reading of property 'name' from undefined. Ensure the user object exists before accessing its properties. Update the condition to check if 'user' and 'user.name' both exist before rendering user.name.